### PR TITLE
feat: add projects section

### PR DESCRIPTION
**Pull request by rpandox**
### Summary of changes

- Add Projects section
- Make the projects section Responsive
- Use iconify package insted of adding the icons files
- Update _config.yml file 
- Add _projects directory

### Outputs

Screenshot 2024-10-20 at 7 08 16 PM

Screenshot 2024-10-20 at 7 09 05 PM

### CODE REVIEW

File: _includes/projects_section.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        >
+        {% endfor %}
+      </div>
+    </div>
+
+    <!-- Tags Section -->
+    <div class="flex flex-wrap gap-2">
+      {% for tag in project.tags %}
+      <span
+        class="bg-[{{ site.bg-colors.softBlue }}] text-[{{ site.text-colors.blue }}] px-3 py-1 rounded-full text-sm font-bold"
```
Comment (prabhavpokharel) [inline _includes/projects_section.html:41]: I suggest not to use two different types of blue color here. The same color with different intensity or opacity will do.

**Review by BISHALTWR — APPROVED** [review]
I suggest adding some more dummy projects. 
And we will need to add a project page soon.
It looks good for now ✨

File: _includes/projects_section.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
+<div class="flex flex-col justify-center gap-3 py-12 mt-4">
+  <h2
+    class="text-5xl font-bold text-[{{ site.text-colors.darkblue }}] text-center mb-8"
+  >
+    Projects
+  </h2>
+
+  <div
+    class="w-[80vw] mx-auto bg-[{{ site.bg-colors.gray }}] p-6 rounded-2xl border border-[{{ site.bg-colors.skyBlue }}] space-y-6"
```
Comment (prabhavpokharel) [inline _includes/projects_section.html:9]: I prefer `space-y-3` here.

File: _includes/projects_section.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  >
+    {% for project in site.projects limit: 3 %}
+    <!-- Project Card -->
+    <div
+      class="flex flex-col md:flex-row justify-start md:justify-between items-start space-y-6 md:space-y-0"
+    >
+      <!-- Left Section: Icon and Title -->
+      <div class="flex items-center space-x-3">
```
Comment (prabhavpokharel) [inline _includes/projects_section.html:17]: I suggest removing `space-x-3` here!

File: _includes/projects_section.html (hunk 1 above)
Comment (rpandox) [reply _includes/projects_section.html:41]: the colors are seperate because one is for background and for text seperate

**Review by gaurovgiri — CHANGES_REQUESTED** [review]
Create a `tech-logo.yml` file under `_data` and then use it as key value pair to store the correct value for the attribute of the logo. For more clarity, see #112 
https://github.com/computerclubkec/computerclubkec.github.io/pull/112#issuecomment-2414509460

**Review by gaurovgiri — APPROVED** [review]
LGTM!

**Comment by BISHALTWR** [comment]
nice !!

File: _includes/projects_section.html (hunk 1 above)
Comment (prabhavpokharel) [reply _includes/projects_section.html:50]: Yes, you're absolutely correct, so I'm suggesting a different intensity. I liked how you used `bg-green-200 text-green-700` which is the same color but a different shade and also here `bg-orange-200 text-orange-700`. 

Correct me if I am wrong here, @rpandox .

Comment (rpandox) [reply _includes/projects_section.html:50]: Noted. You are right

Get back to you with the changes

Comment (rpandox) [reply _includes/projects_section.html:50]: Noted. You are right

Get back to you with the changes

**Comment by rpandox** [comment]
@prabhavpokharel

**Review by prabhavpokharel — APPROVED** [review]
LGTM!

**Pull request merged**